src/components/Options.jsx

```diff
@@ -24,7 +24,7 @@ function Options() {
 
   useEffect(() => {
     chrome.storage.onChanged.addListener(({ userData }) => {
-      dispatch(updateUser(userData.newValue));
+      dispatch(updateUser(userData?.newValue));
```

Comment:
```suggestion
      if(userData) {
        dispatch(updateUser(userData.newValue));
      }
```
 현재 코드는 dispatch(updateUser(null)); 이 실행되는 상황이 있을 것 같습니다.